README.md

```diff
@@ -2,27 +2,37 @@
 
 A command line task running in an A2 module that can "see" all of the schemas for all of the defined doc and widget types, their sub-array and object fields, et cetera.
 
-Iterates over all of these and prepares a folder with schemas in the A3 format, broken down in a sensible directory structure. You would be required to specify a folder name as the parent for these.
+Iterates over all of these and adds a `schema.js` file in the A3 format in every custom module. You can change the parent folder name of modules. By default, it is `lib/modules`.
 
 The directory structure would look like:
 
+```
 modules
-  default-page
+  custom-module
     schema.js
+  custom-page
+    schema.js
+  custom-widget
+   schema.js
+```
 
 Inside each one the format would be:
 
+```js
 module.exports = (self, options) => {
   return {
     ... a3 fields here
   };
 };
+```
 
-This makes it easy to pull into default-page/index.js:
+This makes it easy to pull into custom-module/index.js:
 
+```js
 module.exports = {
-  fields(self, options) => require('./schemas.js')(self, options)
+  fields(self, options) => require('./schema.js')(self, options)
 };
+```
 
 This greatly helps with A2 to A3 migration although it miss things like fields being optional based on module.
```

Comment:
This greatly helps with A2 to A3 migration although manual review is required because it does not understand:

* Fields being optional based `beforeConstruct` logic
* Fields being inherited from a base class module like `apostrophe-pieces`, i.e. not necessary to redefine them in a subclass
* Fields being removed via `removeFields`
* Fields that no longer generally exist in A3, e.g. `published`